src/app.jl

```diff
@@ -78,59 +106,175 @@ function Base.getproperty(app::DashApp, name::Symbol)
     name == :layout ? getlayout(app) : Base.getfield(app, name)
 end
 
-    
+
 
 
 
 """
     dash(name::String; external_stylesheets ::Vector{String} = Vector{String}(), url_base_pathname::String="/")
```

Comment:
@rpkyle in f4cceac I completely removed the default values from the signature in docstring, leaving them only in the argument description. I think it will be more readable this way.